CHANGELOG.md

```diff
@@ -0,0 +1,38 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+### Added
+- New home page design & content
+- New about page design & content
+- New careers page design & content
+- New locations page design & content
+- Build-time fetching of job listings from SmartRecruiters (via the `job-listings` lib)
+- Animated flourishes
+- Animated homepage hero
+- Client-side JavaScript transpilation and bundling (using Babel and Rollup)
+- JavaScript unit tests using Jest
+- [Cypress](https://www.cypress.io/) integration tests
+- [Pa11y](http://pa11y.org/) accessibility tests
+- ESLint with Prettier
+- Templated `robots.txt` that can be configured per deployment environment
+- Image optimisation task
+
```

Comment:
... `- A crap-ton of documentation`

Sorry, that's my bad. I should've added that as part of my PR. On it.